DESCRIPTION

```diff
@@ -11,7 +11,8 @@ Imports:
     RPostgres,
     vaultr,
     whisker,
-    yaml
+    yaml,
+    dplyr
```

Comment:
totally minor, but can you order dependencies alphabetically please (but capitals first) - so this goes just above `vaultr`